client/src/components/SectionTable/SectionTable.js

```diff
@@ -79,6 +80,28 @@ class SectionTable extends PureComponent {
                         <Fragment />
                     )}
                 </div>
+
+                <div
+                    style={{
+                        display: 'inline-flex',
+                        marginTop: '0px',
+                        fontSize: '15px',
+                    }}
+                >
+                    <a
+                        target="blank"
+                        href={
+                            'https://zotistics.com/?&selectQuarter=&selectYear=&selectDep=' +
+                            urlEncode +
+                            '&classNum=' +
+                            this.props.courseDetails.courseNumber +
+                            '&code=&submit=Submit'
```

Comment:
@Maybe14 Can you ping our Discord channel when the URL params are fixed, so we can come back later and remove the unnecessary ones?

Reply:
Note: this doesn't block the PR, we can add this to the "Future Follow" section